wandb/sdk/artifacts/artifact.py

```diff
@@ -1131,6 +1131,7 @@ def add_file(
         local_path: str,
         name: Optional[str] = None,
         is_tmp: Optional[bool] = False,
+        skip_cache: Optional[bool] = False,
         policy: Optional[Literal["mutable", "immutable"]] = "mutable",
```

Comment:
based on the doc-string it is not optional, Optional in typing means that it could accept None, see here: https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
you didn't actually change the types or explained why it is in fact Optional